uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/BAO/ContactWorkPattern.php

```diff
@@ -188,4 +190,38 @@ public static function getWorkDayType($contactId, DateTime $date) {
     return $workDayTypeId;
   }
 
+  /**
+   * Returns a list of ContactWorkPattern instances for the given $contactID,
+   * which overlap the period enclosed by the given $start and $end dates
+   *
+   * @param int $contactID
+   * @param \DateTime $start
+   * @param \DateTime $end
+   *
+   * @return \CRM_HRLeaveAndAbsences_BAO_ContactWorkPattern[]
```

Comment:
Even when it's empty, it's still an array of ContactWorkPatterns, just without any elements. Also, if you check here https://phpdoc.org/docs/latest/references/phpdoc/types.html#arrays you'll see that when no type is specified, then the correct usage is `array` and not `[]`